ansible/setup.yml

```diff
@@ -103,10 +103,10 @@
         creates: "/usr/local/lib/libsodium.so.{{ libsodium_module_version }}"
       tags:
         - libsodium
+      register: installed
 
     - name: Run ldconfig
       command: ldconfig
-      args:
-        creates: /etc/ld.so.cache
       tags:
         - libsodium
+      when: installed
```

Comment:
If you keep it just for workaround please add a comment that it's an workaround. This change itself is meaningless otherwise because the variable always exists (it's a dict): http://docs.ansible.com/ansible/latest/common_return_values.html

- So, you either check for installed.rc (to have meaning)
- add a comment that it's a linter workaround (no meaning)

Reply:
workaround commend added